leaderelection/chaosduck/main.go

```diff
@@ -116,8 +117,7 @@ func quack(ctx context.Context, kc kubernetes.Interface, component string, leade
 }
 
 func main() {
-	ctx := signals.NewContext()
-	ctx = sharedmain.EnableInjectionOrDie(ctx, nil)
+	ctx, _ := injection.EnableInjectionOrDie(signals.NewContext(), nil)
```

Comment:
If we have to do this, that means we are really trying to avoid calling the original function. I would recommend to have a roadmap to deprecate the original function.

Reply:
It is marked deprecated and documented redirected to the new one version.

Reply:
That's awesome. Is this on purpose not to call the start function here? Note that the original logic will have informers started here.

Reply:
I think this might skip your attention. Otherwise LGTM.

Reply:
I checked and the duck does not use informers so it is not needed.

Reply:
Thanks for checking.